ko/admin/admin_utils.rst

```diff
@@ -3186,6 +3191,89 @@ paramdump
 
         cubrid paramdump -C demodb
 
+.. _tde-utility:
+
+tde
+---
+
+**cubrid tde** 유틸리티는 데이터베이스의 TDE 암호화를 관리하기 위하여 사용되며, **DBA** 사용자만 수행할 수 있다. **cubrid tde** 유틸리티를 사용하면 데이터베이스에 등록된 키를 변경할 수 있고, 키 파일에 새로운 키를 안정적으로 추가하고 제거할 수 있다. 또한, 지금까지 키 파일에 추가된 키들과 데이터베이스에 등록된 키가 무엇인지 조회할 수 있다. 자세한 내용은 :ref:`tde` 을 참고한다.
```

Comment:
I have not used 마스터 키  or 데이터 키 out of TDE Section in security.rst intentionally because 데이터 키 is not exposed to the user actually. Do you think this makes it more confused?

Reply:
I understand your intention. And too many places refer to the master key as a key. I don't think it is necessary to modify.